### DISCUSSION TITLE
How to deploy for Windows if I use a Mac for development?

### DISCUSSION THREAD

**Discussion by hmihail** [AvaloniaUI/Avalonia#14224] [Q&A] [answered]
I want to write a small .Net 8 application (to be used on Windows and MacOS) using AvaloniaUI. My dev machine is a Mac, and my question is how do I build the app for Windows? Do I need to open the app on a Windows machine and built it there?

**Comment by maxkatz6** [comment, marked as answer]
You can build your app on macOS just fine. 
In general, it's as simple as `dotnet publish MyProject.csproj -r win-x64`.

Before, .NET SDK had a bug that ".exe" file was missing embed resources, typically app icon. But with .NET 8 it shouldn't be an issue. https://github.com/dotnet/runtime/issues/3828

Keep in mind, though, macOS is still not a Windows, not all tools are available. And it might be problematic to create an installer there or sign the app.

**Reply by hmihail** [reply]
Thanks! I have an older Windows machine available, I will do the final build and packaging there. I just wanted to know how to make quick test builds.
